_episodes/11-wrap-up.md

```diff
@@ -7,9 +7,9 @@ questions:
 objectives:
 - "Recap what you've learned"
 keypoints:
-- "A summary of everything so far"
+- “Your own summary of everything so far"
 ---
 
-## Markdown again
+## Sum everything up for yourself.
```

Comment:
Nah it just came to me - glad you approve! I'm thinking we could definitely get a Google form on there. I'll make a draft and send it your way. I figure it can't hurt to have an extra survey for this specific module even if the whole ReproNim thing has one, right? As for the pull request stuff, I'll brainstorm a bit to see how to get that point across but I'm not sure a simple sentence would be enough, i.e. it wouldn't be effective. What do you think?

Reply:
@Ninja Do we have "Exit surveys" for every topic?  it would be nice to link them here.

Reply:
@chrispycheng Let's just wait for now, we could do it later and meanwhile finalize the fixes/edits